validator/coordinator/src/test/java/tech/pegasys/artemis/validator/coordinator/Eth1DataCacheTest.java

```diff
@@ -288,6 +288,30 @@ void pruneAfterGenesis() {
     assertThat(eth1DataCache.getMapForTesting().values()).containsExactly(eth1Data3);
   }
 
+  @Test
+  void pruneAllBlockData() {
+    eth1DataCache.startBeaconChainMode(genesisState);
+    eth1DataCache.onSlot(new SlotEvent(START_SLOT));
+
+    // First two Eth1Data timestamps inside the spec range for this voting period
+    CacheEth1BlockEvent cacheEth1BlockEvent1 =
+        createRandomCacheEth1BlockEvent(UnsignedLong.valueOf(359));
+    CacheEth1BlockEvent cacheEth1BlockEvent2 =
+        createRandomCacheEth1BlockEvent(UnsignedLong.valueOf(360));
+
+    // This Eth1Data timestamp inside the spec range for the next voting period
```

Comment:
The comment here is misleading. This cacheEth1BlockEvent3 is still inside the current voting period, and that's why it gets cleaned up.